### ISSUE TITLE
UNMET PEER DEPENDENCY

### ISSUE THREAD

**Issue by connercowling** [graphql/express-graphql#393]
```
...
"dependencies": {
    "express": "^4.15.4",
    "express-graphql": "^0.6.11"
  },
  "devDependencies": {
    "babel-cli": "^6.26.0",
    "babel-core": "^6.26.0",
    "babel-preset-env": "^1.6.0"
  }
...
```
and

`npm install`

produces 

```
...
├─┬ express-graphql@0.6.11
│ ├── http-errors@1.6.2
│ └─┬ raw-body@2.3.1
│   ├── bytes@3.0.0
│   └── iconv-lite@0.4.18
└── UNMET PEER DEPENDENCY graphql@^0.10.0 || ^0.11.0
```

**Comment by vjwilson** [comment]
@connercowling ,

This problem came up on a recent episode of Fun Fun Function (https://www.youtube.com/watch?v=lAJWHHUz8_8). The solution there was to manually install the GraphQL package as a dependency (not dev):

```
npm install --save graphql
```

Hope that helps.

**Comment by IvanGoncharov** [comment]
@connercowling @vjwilson 📦 v0.6.12 released with support for graphql v0.12.x and v0.13.x.
